src/Tests/WebParts/AddPnPWebPartToWikiPageTests.cs

```diff
@@ -77,7 +77,7 @@ public void AddPnPWebPartToWikiPageTest()
 				// This is a mandatory parameter
 				// From Cmdlet Help: Column number where the web part must be placed
 				var column = "";
-				// From Cmdlet Help: Must there be a extra space between the web part
+				// From Cmdlet Help: Must there be an extra space between the web part
```

Comment:
Thanks for correcting this.
Since we don't use this project anymore, please don't make any changes in the tests project in the future.

Reply:
Okay, I simply changed it back.